### PR TITLE
Feat/share profile

### PR DESCRIPTION
**Pull request by boidushya**
# Description

Profile Modal

# Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Examples/Screenshots (Optional)

| ENS | Dark       | Light        |
| -- | ------------ | ------------ |
| Yep | image | image |
| Nope | image | image |

### Share ↔️ Profile

https://user-images.githubusercontent.com/26746725/233225611-d28c52ed-e1ba-4624-82dd-876fd25569c4.mov

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/account/Profile/index.tsx [hunk 1]
```diff
@@ -1,11 +1,232 @@
...
+              localStorage.removeItem('ens-records')
+            }}
+          >
+            <CrossIcon />
+          </button>
+        </div>
+        <div className="Profile__TitleBar__background__container">
+          <div className="Profile__TitleBar__background" />
+        </div>
+        <div className="Profile__icon">{ensAvatar && <img src={ensAvatar} alt="ENS Avatar" />}</div>
```
Comment (devceline) [inline src/components/account/Profile/index.tsx:107]: Couldn't we use `Avatar` component for this? As it'd generate unique background gradients for each addres?

**Review by devceline — COMMENTED** [review]
This is fantastic, one minor comment regarding avatar and then it's all good to go

File: src/components/account/Profile/index.tsx (hunk 1 above)
Comment (boidushya) [reply src/components/account/Profile/index.tsx:107]: Right!! I didnt realise we had that, mb

**Review by devceline — APPROVED** [review]
LGTM, :ship: it

**Pull request merged**